ministryofjustice/peoplefinder
# CDPT-1193 external user access

## Description

Give external users read only access

## Self-review checklist


### Screenshots

N/A

### Related JIRA tickets

https://dsdmoj.atlassian.net/browse/CDPT-1711

### Deployment


### Manual testing instructions

Once deployed to dev and staging environments test external user accounts will need to be added.
When these have been added logon under an external account and see if you can Add/Delete/Edit profiles or teams.
You should only be able to view profiles or teams.
